docs/DeveloperGuide.md

```diff
@@ -385,6 +388,145 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+**Trigger:** EstateEase validates the entered details and detects that the block number is missing when the housing type is HDB/Condo.
+
+**Precondition for Extension 3k and 3l:** EstateEase has received the details of the home from the user. <br>
+**Trigger:** EstateEase validates the entered details and detects that the unit number is missing when the housing type is HDB/Condo.
+
+**Extensions**
+* 1a. The contact list does not have any home-seller. <br>
+    * 1a1. EstateEase shows an error message stating that the contact list does not have home-seller. <br>
+      Use case ends.
+
+* 3a. EstateEase detects missing name of the home-seller in the entered data. <br>
+    * 3a1. EstateEase shows an error message regarding missing name. <br>
+      Use case resumes from step 2.
+
+* 3b. EstateEase detects invalid name of the home-seller in the entered data. <br>
+    * 3b1. EstateEase shows an error message regarding invalid name. <br>
+      Use case ends.
+
+* 3c. EstateEase detects that the name does not belong to home-seller, but to home-buyer instead. <br>
+    * 3c1. EstateEase shows an error message regarding home can only be attached to home-seller, instead of home-buyer. <br>
+      Use case resumes from step 2.
+
+* 3d. EstateEase detects missing housing type in the entered data. <br>
+    * 3d1. EstateEase shows an error message regarding missing housing type. <br>
+      Use case resumes from step 2.
+
+* 3e. EstateEase detects incorrect housing type in the entered data. <br>
+    * 3e1. EstateEase shows an error message regarding the entry of incorrect of housing type. <br>
+      Use case resumes from step 2.
+
+* 3f. EstateEase detects missing block number in the entered data. <br>
+    * 3f1. EstateEase shows an error message regarding missing block number. <br>
+         Use case resumes from step 2.
+
+* 3g. EstateEase detects missing street name in the entered data. <br>
+    * 3g1. EstateEase shows an error message regarding missing street name. <br>
+      Use case resumes from step 2.
+
+* 3h. EstateEase detects missing unit number in the entered data. <br>
+    * 3h1. EstateEase shows an error message regarding missing unit number. <br>
+      Use case resumes from step 2.
+
+* 3i. EstateEase detects incorrect format for unit number in the entered data. <br>
+    * 3i1. EstateEase shows an error message regarding incorrect format for unit number. <br>
+      Use case resumes from step 2.
+
+* 3j. EstateEase detects missing postal code in the entered data. <br>
+    * 3j1. EstateEase shows an error message regarding missing postal code. <br>
+      Use case resumes from step 2.
+
+* 3k. EstateEase detects incorrect format for postal code in the entered data. <br>
+    * 3k1. EstateEase shows an error message regarding incorrect format for postal code. <br>
+      Use case resumes from step 2.
+
+**Use case: UC00 - Viewing of homes based on home-seller's priority**
```

Comment:
Change the UC# here too

Reply:
Yeap updated the numbering of the use case. Thanks for reminding.